analytic_engine/src/instance/close.rs

```diff
@@ -67,6 +70,7 @@ impl Closer {
         let removed_table = self.space.remove_table(&request.table_name);
         assert!(removed_table.is_some());
```

Comment:
Why not just call `unwrap` or `expect`?

Reply:
I guess there is no significant different between the two ways.